controllers/topics/index.js

```diff
@@ -221,7 +225,8 @@ const followTopicV2 = async (req, res) => {
             secondDetailUser.user_id,
             name,
             detailTokenUser.is_anonymous,
-            with_system_message
+            with_system_message,
+            prevUserToken
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Again, the addition of `prevUserToken` as a parameter to the function call seems fine. Ensure that the called function has been updated to handle this new argument.
```diff
+           prevUserToken
           )
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->